Sources/Converters.swift

```diff
@@ -18,7 +18,8 @@ enum ConvWeightArray {
 
 // MARK: Convolutions
 
-@objc private class ConvDataSource: NSObject, MPSCNNConvolutionDataSource {
+@available(iOS 11.0, macOS 10.13.0, tvOS 11.0, *)
```

Comment:
From the current podspec:
```
  s.ios.deployment_target = '11.0'
  s.osx.deployment_target = '10.13'
```
Looks like this and some other `@available` checks are redundant